src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_MAX_CAPACITY = 10;
+    private final Object[] keys;
```

Comment:
Let's use generic type here
```suggestion
    private final K[] keys;
```